pmlc/pmlc-opt.cc

```diff
@@ -11,6 +11,9 @@
 #include "llvm/Support/SourceMgr.h"
 #include "llvm/Support/ToolOutputFile.h"
 
```

Comment:
We may need to add more tools, like a cpu runner so maybe we should keep pmlc-opt at least under tools dir. Was the  tools/pmcl-opt/ problematic?

Reply:
I was following tensorflow's lead, that's basically how their directory structure looks.

There's actually a CPU runner built into MLIR that we could possibly use, provided we lower into LLVM dialect.

Reply:
Yeah, maybe there is something we can reuse but we would need our own if we want the input to start from Stripe and apply transformations that are not in MLIR core.